lib/install/management/configure.go

```diff
@@ -111,6 +111,11 @@ func (d *Dispatcher) Configure(vch *vm.VirtualMachine, conf *config.VirtualConta
 	}
 
 	if err = d.update(conf, settings, isConfigureOp); err == nil {
+		if conf.ShouldGrantPerms() {
+			if err = GrantOpsUserPerms(d.ctx, d.session.Vim25(), conf); err != nil {
```

Comment:
Not going to block the PR for it, but I find the uncollapsed form of this is significantly easier to visually parse for important operations; the collapsed form needs extra effort to tell if there's a significant side effect or if it's just a test for existing state.
Only time I think it's really necessary to use the collapsed form is when scoping of the return is important:
```go
err := GrantOpsUserPerms(d.ctx, d.session.Vim25(), conf)
if err != nil {
	return errors.Errorf("Cannot init ops-user permissions, failure: %s. Exiting...", err)
}
```